.travis.yml

```diff
@@ -6,7 +6,7 @@ env:
 
 # keep this blank to make sure there are no before_install steps
 before_install:
-
+  npm install codecov
```

Comment:
For me either case is fine, since current way's my preferences. I'll update PR shortly.